BREEZE.rst

```diff
@@ -265,6 +265,40 @@ You should set up the autocomplete option automatically by running:
 
 You get the auto-completion working when you re-enter the shell.
 
+Breeze on Windows
+-----------------
+In Windows environment, you will need to use **pipx** to install Breeze2.
+
+If you are on PowerShell or CMD, use:
+
+.. code-block:: bash
+
+   pipx install -e dev\breeze
+
+If you are using Linux/Bash (POSIX) on Windows, like Git Bash use:
+
+.. code-block:: bash
+
+   pipx install -e dev/breeze
```

Comment:
Forward slash works everywhere, including Powershell and cmd.exe, so we don’t need two code blocks here.

Reply:
does it :) ? Did they learn ?

Reply:
Yes, they learned. `/` works for PowerShell, cmd, and Git Bash. Thank you I will fix it.